docs/source/openshift/exposing-services.adoc

```diff
@@ -59,8 +59,26 @@ $ mysql --user=root --password=admin --host=$(minishift ip) --port=30907
 [[port-forwarding]]
 == Port Forwarding
 
+=== Using `oc port-forward`
 If you want to quickly access a port of a specific pod of your cluster, you can also use the `oc port-forward` command of the xref:../openshift/openshift-client-binary.adoc#[OpenShift CLI].
 
 ----
 $ oc port-forward POD [LOCAL_PORT:]REMOTE_PORT
 ----
+
+=== Using VirtualBox tools
+In case you're using the VirtualBox driver plugin there is another method you can
```

Comment:
Apologies if my previous comment was unclear -- I meant that we need each individual sentence on its own line. :)

```adoc
In case you're using the VirtualBox driver plugin there is another method you can use for port forwarding.
This method will allow for permanent port forwarding as well as for forwarding multiple ports at the same time.
This method requires you to set up a `nodePort` as outlined above.
```